src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maxAllowedNumber = 100;
```

Comment:
```suggestion
    private final int MAX_BALL_NUMBER = 100;
```

Reply:
I think you missed one keyword for constant

Reply:
<img width="1576" alt="Screenshot 2023-08-08 at 14 36 05" src="https://github.com/mate-academy/jv-lottery/assets/61597780/78e4a0e1-d8c8-4090-8a53-ef568ed799ef">

Reply:
> I think you missed one keyword for constant

iIdon't get it, I wrote the 'final' keyword, correct me if I am wrong, please